backend/lib/piece-indexer-service.js

```diff
@@ -30,7 +30,7 @@ export const getDealPayloadCid = async (providerId, pieceCid) => {
       if (parsedPixResponse.error === 'PROVIDER_OR_PIECE_NOT_FOUND') {
         return null
       }
-    } catch {}
+    } catch { }
```

Comment:
Should we throw or log error here?

Reply:
No, as this is checking if the response is an error. If it's not, we parse the response again, below (where we would throw)